scripts/ci/build.sh

```diff
@@ -4,7 +4,7 @@ set -e
 cd "$(dirname "$0")"/../..
 ./bootstrap
 ./configure \
-	--host=${TARGET_ARCH} --build=x86_64-linux-gnu \
+	--host=${TARGET_ARCH} --build=${BUILD_ARCH:-x86_64-linux-gnu} \
```

Comment:
The `if [ -z "$TARGET_ARCH" ]` line below still prevents running the dynamically linked example application on arm64 hosts.

You could add a ${BUILD_ARCH} = ${TARGET_ARCH} comparison to fix this.